src/components/IconButtonWithTooltip.vue

```diff
@@ -20,18 +20,14 @@
 }
 </style>
 
-<script lang="ts">
-import { defineComponent } from "vue";
-export default defineComponent({
-  props: {
-    title: {
-      type: String,
-      required: true,
-    },
-    icon: {
-      type: String,
-      required: true,
-    },
-  },
-})
+<script lang="ts" setup>
+const props = defineProps<{
+  title: string,
+  icon: string,
+  variant?: NonNullable<"flat" | "text" | "elevated" | "tonal" | "outlined" | "plain"> | undefined,
```

Comment:
Do there exists actually types for this? I am fine with this, but Vue should have some kind of type for this?

Reply:
I created #20 to resolve this